packages/storage-plus/src/keys.rs

```diff
@@ -362,15 +421,67 @@ mod test {
         let k: U64Key = 134u64.into();
         let path = k.key();
         assert_eq!(1, path.len());
-        assert_eq!(134u64.to_be_bytes().to_vec(), path[0].to_vec());
+        assert_eq!(134u64.to_be_bytes(), path[0].as_ref());
```

Comment:
Huh?

Isn't `134u64.to_be_bytes()` type `[u8; 8]`
while `path[0].as_ref()` type `&[u8]`?

(pointing to the equal underlying bytes however)